test/tools/OpenCover/OpenCover.psm1

```diff
@@ -359,22 +332,15 @@ function Invoke-OpenCover
     [CmdletBinding(SupportsShouldProcess=$true)]
     param ( 
         [parameter()]$OutputLog = "$pwd/OpenCover.xml",
-        [parameter(Mandatory=$true)]$TestDirectory,
-        [parameter()]$OpenCoverPath,
-        [parameter(Mandatory=$true)]$PowerShellExeDirectory,
+        [parameter()]$TestDirectory = "$($script:psRepoPath)/test/powershell",
+        [parameter()]$OpenCoverPath = "~/OpenCover",
+        [parameter()]$PowerShellExeDirectory = "$($script:psRepoPath)/src/powershell-win-core/bin/debug/netcoreapp1.0/win10-x64",
```

Comment:
>"$($script:psRepoPath)/src/powershell-win-core/bin/debug/netcoreapp1.0/win10-x64", [](start = 47, length = 82)

This should be "$($script:psRepoPath)/src/powershell-win-core/bin/CodeCoverage/netcoreapp1.0/win10-x64"